src/styles/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  width: $card-width;
```

Comment:
Ensure that the variable `$card-width` is defined in your SCSS files. If it's not defined, it will cause an error during compilation.